README.md

```diff
@@ -118,7 +118,18 @@ spec:
 
 #### TLS
 
-TLS is optional but recommended for production deployments. It is used if `TLS_KEY` and `TLS_CERT` environment variables are set.
+TLS is optional but recommended for production deployments. It is used if `TLS_KEY` and `TLS_CERT` environment variables are set. Example:
+
+```bash
+docker run -p 8080:8080 -e TLS_KEY="$(cat sourcegraph.key)" -e TLS_CERT="$(cat sourcegraph.crt)" sourcegraph/lang-typescript
+```
```

Comment:
This is inside the "Using Kubernetes" section - could you add the docker steps in a new TLS section under "Using Docker"?

Reply:
Thanks, moved under the Docker section